### PR TITLE
feat(percentage): add new phrases

### PR DESCRIPTION
**Pull request by decimalcubed**
grr

### CODE REVIEW

**Review by sirjahames — COMMENTED** [review]
Seems good @quantix-dev

**Comment by decimalcubed** [comment]
accept my pr

File: src/interactions/fun/percentage.ts [hunk 1]
```diff
@@ -51,6 +51,38 @@ export class percentage extends QInteraction {
...
+                            name: "British",
+                            value: "🇬🇧"
+                        },
+                        {
+                            name: "Rizz",
+                            value: "a rizz master"
+                        },
+                        {
+                            name: "Swag",
+                            value: "<a:cool~1:937421846842843257>"
```
Comment (quantix-dev) [inline src/interactions/fun/percentage.ts]: Change this to `<a:cool:937421846842843257>` so the emoji actually loads

**Review by quantix-dev — CHANGES_REQUESTED** [review]

**Review by AnthonyFuller — APPROVED** [review]

**Review by quantix-dev — APPROVED** [review]
Changes will be deployed later today

**Pull request merged**